weatherApp/Features/WeatherList/UI/CurrentWeatherListPresenter.swift

```diff
@@ -9,12 +9,15 @@
 class CurrentWeatherListPresenter {
     
     private var currentWeatherList: [CurrentWeatherViewModel] = []
+    private var currentLocationWeather: CurrentWeatherViewModel?
     private let weatherRepository: WeatherRepository
     private let navigationService: NavigationService
+    private let locationService: LocationService
```

Comment:
Can you create a protocol for LocationService? This will be very helpful later if you create some unit tests.